authors/apps/authentication/views.py

```diff
@@ -25,8 +25,8 @@ def post(self, request):
         serializer = self.serializer_class(data=user)
         serializer.is_valid(raise_exception=True)
         serializer.save()
-
-        return Response(serializer.data, status=status.HTTP_201_CREATED)
+        token = serializer.instance.token
+        return Response({'token': token}, status=status.HTTP_201_CREATED)
```

Comment:
Please include a success message along with the token. Also do this for `line 48`